### PR TITLE
fix/Electrum playwright test

### PR DESCRIPTION
**Pull request by ondracja**
- fixed and refactored the `electrum` test
- adjusted other tests to match 
- added new tests to CI yaml file

### CODE REVIEW

File: packages/suite-desktop-core/e2e/support/common.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ export const launchSuite = async () => {
 export const waitForDataTestSelector = (window: Page, selector: string, options = {}) =>
     window.waitForSelector(`[data-test="${selector}"]`, options);
 
+// TODO: usage of .click is discouraged by playwright devs. Refactor all usage to locator.click()
```
Comment (karliatto) [inline packages/suite-desktop-core/e2e/support/common.ts:43]: Why not changing it now in this function? It is just one line, right?

File: packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            await settingsHeader.click();
+        }
+        await window.getByTestId('@settings/menu/debug').waitFor({ state: 'visible' });
+    }
+
+    async goToDesiredSettingsPlace(
+        window: Page,
+        desiredLocation: 'debug' | 'general' | 'device' | 'wallet',
+    ) {
+        let desiredLocationWaiter: string;
```
Comment (karliatto) [inline packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts]: NIT: Why call it `...Waiter`? It seams to me better naming `desiredLocationDataTest`

File: packages/suite-desktop-core/e2e/support/common.ts (hunk 1 above)
Comment (ondracja) [reply packages/suite-desktop-core/e2e/support/common.ts:43]: After that, I believe that there is no point in having such a function. I'd like to slowly get rid of it during the refactor of current tests (to follow the page action model).

File: packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts (hunk 2 above)
Comment (ondracja) [reply packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts]: Hmm I wanted to make a clear distinction that this is the element that is used for verifying the page that is being oepend. Is it confusing?

**Review by karliatto — APPROVED** [review]
Electrum tests are now passing https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/5657619825 :tada:

File: packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts (hunk 2 above)
Comment (ondracja) [reply packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts]: After a slack discussion, I'll change it to `desiredLocationDataTest`.

File: ci/test.yml [hunk 3]
```diff
@@ -176,7 +176,7 @@ suite web canary fws manual:
     matrix:
       - TEST_FILE: ["spawn-tor", "spawn-bridge"]
         CONTAINERS: "trezor-user-env-unix"
-      - TEST_FILE: ["electrum"]
+      - TEST_FILE: ["electrum", "bug-report"]
```
Comment (mroz22) [inline ci/test.yml]: does bug-report test need electrum regtest container?

**Review by mroz22 — CHANGES_REQUESTED** [review]

File: ci/test.yml (hunk 3 above)
Comment (mroz22) [reply ci/test.yml]: if not, it should be included on line 178. this is the mechanism used not to run unneeded containers

Comment (mroz22) [reply ci/test.yml]: otherwise good job, feel free to resolve my review in case I don't react :P

Comment (ondracja) [reply ci/test.yml]: nope, I just wanted to add it to the run. Should I move it to the upper `TEST_FILE` ?

**Review by mroz22 — APPROVED** [review]

**Pull request merged**